### ISSUE TITLE
Should : A parameter cannot be found that matches parameter name 'BeLikeExactly'

### ISSUE THREAD

**Issue by SQLDBAWithABeard** [pester/Pester#966]
## 1. Provide a general summary of the issue in the Title above

I am getting this error when trying to use the BeLikeExactly Assertion in 4.1.1 or 4.0.8

Error - Should : A parameter cannot be found that matches parameter name 'BeLikeExactly'

## 2. Describe Your Environment

Operating System, Pester version and PowerShell version:

Pester version     : 4.1.1 4.0.8 
WindowsPowerShell\Modules\Pester\4.1.1\Pester.psm1 WindowsPowerShell\Modules\Pester\4.0.8\Pester.psm1
PowerShell version : 5.0.10586.1007
OS version         : Microsoft Windows NT 10.0.10586.0

## 4.Current Behavior
```
$actual | Should -BeLikeExactly "Actual *"
Should : A parameter cannot be found that matches parameter name 'BeLikeExactly'.
At line:1 char:18
+ $actual | Should -BeLikeExactly "Actual *"
+                  ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Should], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Should
```

**Comment by nohwnd** [comment]
That is because it was not registered prior 4.2.0-alpha, [see here](https://github.com/pester/Pester/blob/7c8a7f99463b998aeb4a6965427a2f2aebdbba96/Functions/Assertions/BeLikeExactly.ps1) the Add-ShouldParameter is missing.

**Comment by it-praktyk** [comment]
Is it a bug or a feature? ;-)

Can we close it now?

**Comment by nohwnd** [comment]
fixed in 4.2.0
